include/usgscsm/UsgsAstroLsSensorModel.h

```diff
@@ -971,6 +971,12 @@ class UsgsAstroLsSensorModel : public csm::RasterGM,
   // Compute the determinant of a 3x3 matrix
   double determinant3x3(double mat[9]) const;
 
+  // Apply a rotation and translation to a state string. The effect is
+  // to transform the position and orientation of the camera in ECEF
+  // coordinates.
+ void applyTransformToState(ale::Rotation const& r, ale::Vec3d const& t,
```

Comment:
Since this isn't operating on the actual model's state this should be a static method.